packages/fhir-converter/src/templates/cda/Resources/Observation.hbs

```diff
@@ -63,6 +62,12 @@
         {{!-- in some cases text is here in the b64 field. Adding so no data loss but not sure if thats the way its supposed to be. also how it renders I do not know         --}}
         {{else if observationEntry.value._b64}}
             "valueString":"{{observationEntry.value._b64}}",
+        {{else if observationEntry.value.translation.value}}
+            {{#if observationEntry.value.translation.originalText._}}
+                "valueString":"{{observationEntry.value.translation.value}} {{observationEntry.value.translation.originalText._}}"
```

Comment:
what is this syntax where two {{}} {{}} sets of objects?

Reply:
String concatenation for `unit + value`